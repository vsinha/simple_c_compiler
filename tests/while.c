
long i;

void main()
{
	i = 0;
	
	while (i<9) {
		printf("i=%d\n", i);
		i=i+1;
	}

	printf("OK\n");
}

