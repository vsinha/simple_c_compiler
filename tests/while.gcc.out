while.c: In function 'main':
while.c:9:3: warning: incompatible implicit declaration of built-in function 'printf' [enabled by default]
while.c:9:3: warning: format '%d' expects argument of type 'int', but argument 2 has type 'long int' [-Wformat]
while.c:13:2: warning: incompatible implicit declaration of built-in function 'printf' [enabled by default]
